packages/frontend-2/components/model/DashboardList.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <section>
+    <div class="flex items-center justify-between gap-4 mt-8 mb-4">
+      <div class="flex items-center gap-4">
+        <h1 class="h4 text-foreground font-bold flex items-center">
+          {{ title }}
+          <span class="ml-2 caption">{{ numModels }}</span>
+        </h1>
+      </div>
+      <div v-if="enableSearch">
+        <FormTextInput
+          name=""
+          label=""
+          placeholder="search"
+          class="bg-background shadow hover:shadow-lg"
+        />
+      </div>
+    </div>
+    <div class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-4">
+      <ModelCard
+        v-for="model in models.splice(0, numModels)"
```

Comment:
`splice()` actually mutates the source array, is that really what you're going for here? maybe you meant to use `slice()` instead?